packages/zowe-explorer/__tests__/__unit__/uss/ZoweUSSNode.unit.test.ts

```diff
@@ -1386,84 +1386,6 @@ describe("ZoweUSSNode Unit Tests - Function node.openedDocumentInstance()", () =
     });
 });
 
-describe("ZoweUSSNode Unit Tests - Function node.initializeFileOpening()", () => {
```

Comment:
thanks for cleaning up these tests after exporting/moving the function outside the node